README.rst

```diff
@@ -151,6 +150,16 @@ dynamic programming and is linear O(n):
     >>> print(big_o.big_o(fib_dp, big_o.datagen.n_, n_repeats=100, min_n=200, max_n=1000)[0])
     Linear: time = -1.8E-06 + 7.3E-06*n (sec)
 
+Report Generation
+-------
+
+We can generate the a report using the output generated from calling the big_o
+function. The report defines the best time complexity along with the the others 
+estimates. The results are sorted based on the residuals and the rankings are 
+put in place to allow comparisons.
+
+    >>> best, others = big_o.big_o(heapify, data_generator_heapify, max_n=10**7)
+    >>> big_o.print_report(best, others)
```

Comment:
could you please include the output of the report for this command? (not a screenshot, you can put the text output in triple backquotes)